webapp/src/projects.tsx

```diff
@@ -152,7 +152,7 @@ export class Projects extends data.Component<ISettingsProps, ProjectsState> {
                             window.open('https://youtu.be/' + scr.youTubeId, 'yt');
                         else if (/^https:\/\//i.test(scr.url)) // External video
                             window.open(scr.url, '_blank');
-                        else if (scr.url) // Docs url, open in new tab
+                        else if (/^\//i.test(scr.url)) // Docs url, open in new tab
                             window.open(scr.url, '_blank');
                         else
                             this.props.parent.newEmptyProject(scr.name.toLowerCase(), scr.url);
```

Comment:
probably just want to show a warning and ignore the URL.

Reply:
Cool. Done